src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -155,8 +155,22 @@ public async override Task<P.CreateInstanceResponse> StartInstance(P.CreateInsta
             {
                 try
                 {
-                    string instanceId = await this.GetClient(context).StartNewAsync(
-                        request.Name, request.InstanceId, Raw(request.Input));
+                    string instanceId = request.InstanceId ?? Guid.NewGuid().ToString("N");
+                    TaskHubClient taskhubClient = new TaskHubClient(this.GetDurabilityProvider(context));
+
+                    // TODO: Ideally, we'd have a single method in the taskhubClient that can handle both scheduled and non-scheduled starts.
+                    // TODO: the type of `ScheduledStartTimestamp` is not nullable. Can we change it to `DateTime?` in the proto file?
```

Comment:
I believe proto uses its own type: `Timestamp`, which is an object, so it can be `null` (whereas `DateTime` and `DateTimeOffset` are structs). So no, we cannot change it to `DateTime?`.